templates/resolver.go

```diff
@@ -22,9 +22,90 @@ int main() {
 // We could consider using some new Go feature to embed it as a static resource.
 // At the time of creation of this, this is not a priority.
 const RandH = `
+#pragma once
+#include <ctime>
+#include <string>
+enum CASE {
+	LOWERCASE,
+	UPPERCASE,
+	MIXEDCASE
+};
 class Rand {
+private:
+	int seed;
+	void init() {
+		srand(seed);
+	}
+	void readArgs(int argc, char * argv[]) {
```

Comment:
InitSeedFromArgs (Or something like this) is a better name than readArgs which might be a little bit confusing